### ISSUE TITLE
Demo header is missing AI2 brand badge

### ISSUE THREAD

**Issue by aaronsarnat** [allenai/allennlp#684]
After updating http://allennlp.org/ with the new AI2-branded header/nav, Oren pointed out that a similar treatment is missing from the Demo header.

**Comment by aaronsarnat** [comment]
@schmmd I was going to assign this to myself, but don't have privileges. I need to speak to Mark H. about time allocation for this, but by itself shouldn't take too long to implement.

**Comment by schmmd** [comment]
@aaronsarnat if you can't find the time, let's get together for 5 minutes and you can hand back off to us.

**Comment by aaronsarnat** [comment]
Thanks for reminding me. I should be able to help with this or at the very least give some guidance. Oren and I chatted about it briefly last week, and he didn't seem terribly concerned about this after I pointed out there were technical and UX reasons why the demo header is different than the website header. That said, it's probably best to add an AI2 logo somewhere to the demo header to be on the safe side.

With that in mind, what are your thoughts on priority for this?
